### PR TITLE
lapack: enable all msvc builds

### PR DESCRIPTION
**Pull request by NeroBurner**
test all the lapack msvc builds. They all should just download the prebuilt c-library

### CODE REVIEW

**Comment by ruslo** [comment]
> [hunter ** FATAL ERROR **] Broken package:  LAPACK on MSVC is only available on WIN64

Enable only 64 bit toolchains.

**Review by ruslo — CHANGES_REQUESTED** [review]
You can test everything yourself before opening pull request with these instructions:
* https://docs.hunter.sh/en/latest/creating-new/update.html

**Comment by NeroBurner** [comment]
ah, new docs, nice! :+1:

**Comment by NeroBurner** [comment]
I created a branch `test.lapack` with the same changes as this PR and pushed it to https://github.com/NeroBurner/hunter/tree/test.lapack

no automatic builds are starting. Am I understanding the documentation the wrong way?

As a side note to the docs at https://docs.hunter.sh/en/latest/creating-new/update.html#testing

```
[hunter]> git fetch ci
[hunter]> git checkout pkg.hunter_box_1
[hunter]> git checkout -b test.hunter_box_1
[hunter]> git merge pr.hunter_box_1
```

the first `git fetch ci` is a bit unclear. I don't see which repository is meant with `ci`. I suspect it is `ingenue/hunter`. Maybe add a note or hint with `git remote add ci https://github.com/ingenue/hunter`

**Comment by ruslo** [comment]
> no automatic builds are starting. Am I understanding the documentation the wrong way?

I guess you haven't registered your Hunter fork in AppVeyor/Travis.

> the first git fetch ci is a bit unclear. I don't see which repository is meant with ci

Branches/remotes described in previous section. I've added a note:
* https://docs.hunter.sh/en/latest/creating-new/update.html#testing

File: appveyor.yml [hunk 1]
```diff
@@ -73,6 +76,9 @@ environment:
     #- TOOLCHAIN: "vs-14-2015"
     #  PROJECT_DIR: examples\LAPACK-CBLAS
 
+    #- TOOLCHAIN: "vs-14-2015-win64"
```
Comment (ruslo) [inline appveyor.yml]: I don't see this toolchain in original file. To remove.

**Review by ruslo — CHANGES_REQUESTED** [review]

File: appveyor.yml (hunk 1 above)
Comment (NeroBurner) [reply appveyor.yml]: Should I add it in a separate PR? I'd like to test `vs-14-2015 win64` for the whole ceres-toolchain

Comment (NeroBurner) [reply appveyor.yml]: OH, it's the CBLAS test, sorry, will remove

**Comment by NeroBurner** [comment]
ah thanks for the tipp

squashed the commits

**Review by ruslo — APPROVED** [review]

**Pull request merged**